### PR TITLE
new tag for mmlu

### PR DESCRIPTION
**Pull request by mailvijayasingh**

### CODE REVIEW

File: scripts/scheduler/hourly_run.sh [hunk 1]
```diff
@@ -35,6 +35,9 @@ echo "./scripts/scheduler/create_job.sh ./cases/hourly_jax.csv \"\" $TAG HOURLY_
 # Run JAX with new model design
 ./scripts/scheduler/create_job.sh ./cases/hourly_jax_new.csv "" $TAG HOURLY_JAX TPU_COMMONS "TPU_BACKEND_TYPE=jax;NEW_MODEL_DESIGN=True"
 
+# Run JAX with mmlu dataset
+./scripts/scheduler/create_job.sh ./cases/hourly_jax.csv "" $TAG HOURLY_JAX_MMLU TPU_COMMONS "TPU_BACKEND_TYPE=jax"
```
Comment (QiliangCui) [inline scripts/scheduler/hourly_run.sh:39]: this should hourly_jax_mmlu.csv

File: cases/hourly_jax_mmlu.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Device,Model,MaxNumSeqs,MaxNumBatchedTokens,TensorParallelSize,MaxModelLen,Dataset,InputLen,OutputLen,ExpectedETEL,NumPrompts
+v6e-1,meta-llama/Llama-3.1-8B-Instruct,256,1024,1,2048,mmlu,1800,128,,1000
```
[the remark this answers is not in the document]
Comment (QiliangCui) [reply cases/hourly_jax_mmlu.csv:2]: here, I think you also want to add 

```
v6e-8,meta-llama/Llama-3.3-70B-Instruct,128,1024,8,2048,mmlu,1800,128,,1000
```

2 tests right?
